modules/mobile/docs/openapi.yaml

```diff
@@ -548,6 +548,179 @@ paths:
       security:
         - Bearer: []
       summary: /v0/appointments/facility/eligibility
+  /v0/appointments/facilities/{facililty_id}/clinics/{clinic_id}/slots:
```

Comment:
I looked at how vaos is doing this. They seem to use "location" insteand of "facility". I assume those are the same, but it seems odd that they use both location and facility in their urls.

Reply:
that was my understanding as well but the facility seemed to be a better description. I don't think any ids provided are can be anything but "Facilities" so generalizing it to location doesn't seem helpful. it matches better with our other facility endpoints already made as well.